apps/backend/prisma/schema.prisma

```diff
@@ -84,6 +84,11 @@ model Chip {
   ownerEncryptionPublicKey String?
   ownerPsiPublicKeyLink    String?
   ownerUserData            Json? // Nullable JSONB field
+  isLocationChip           Boolean?
+  locationId               String?   @unique
+  locationName             String?
+  locationDescription      String?
+  locationData             Json? // Nullable JSONB field
```

Comment:
Who's the owner of a location chip? Is it the admin that set it up? 

I think for this activation this makes sense, but for a future one these fields are basically repeated in the connection chip. Could be interesting to consider what a generic chip would look like for future activations.

Reply:
I think the owner could be the person who owns the chip but the issuer is the community that issues the chip, and these theoretically could be different